pkgs/development/python-modules/dataframe_image/default.nix

```diff
@@ -0,0 +1,126 @@
+{
+  stdenv,
+  pkgs,
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  python3Packages,
+  texlive,
```

Comment:
I'm not familiar with nixpkgs, if it could, texlive could be marked as optional. It is only used during export notebook contains dataframe to pdf.